.gitignore

```diff
@@ -153,6 +153,7 @@ gradle-app.setting
 
 # # VS Code Specific Java Settings
 # DO NOT REMOVE .classpath and .project
+.factorypath
 .classpath

```

Comment:
Why is this a necessary addition?

Reply:
.factorypath contains environment specific IDE settings. 

Here's what mines contains. Adding this will lead to conflicts as each user will have a different path in the EXTJAR paths.

```
<factorypath>
    <factorypathentry kind="EXTJAR" id="/home/ember/.gradle/caches/modules-2/files-2.1/org.littletonrobotics.akit.junction/junction-autolog/3.0.0/abaf89ad1ac7acc95dbc0d3f0785a39acf98dd0e/junction-autolog-3.0.0.jar" enabled="true" runInBatchMode="false"/>
    <factorypathentry kind="EXTJAR" id="/home/ember/.gradle/caches/modules-2/files-2.1/com.squareup/javapoet/1.13.0/d6562d385049f35eb50403fa86bb11cce76b866a/javapoet-1.13.0.jar" enabled="true" runInBatchMode="false"/></factorypath>
```

Reply:
Ack